docs/source/delta-clustering.md

```diff
@@ -0,0 +1,111 @@
+
+## What is liquid clustering used for?
+
+The following are examples of scenarios that benefit from clustering:
+
+- Tables often filtered by high cardinality columns.
+- Tables with significant skew in data distribution.
+- Tables that grow quickly and require maintenance and tuning effort.
+- Tables with access patterns that change over time.
+- Tables where a typical partition key could leave the table with too many or too few partitions.
+
+## Enable liquid clustering
+
+You must enable liquid clustering when first creating a table. Clustering is not compatible with partitioning or `ZORDER`. Once enabled, run `OPTIMIZE` jobs as normal to incrementally cluster data. See [_](#optimize).
+
+To enable liquid clustering, add the `CLUSTER BY` phrase to a table creation statement, as in the examples below:
+
+```sql
+-- Create an empty table
+CREATE TABLE table1(col0 int, col1 string) USING DELTA CLUSTER BY (col0);
+
+-- Using a CTAS statement
+CREATE TABLE table2 CLUSTER BY (col0)  -- specify clustering after table name, not in subquery
+AS SELECT * FROM table1;
+```
+
+.. warning:: Tables created with liquid clustering enabled have Clustering table feature enabled at creation and use Delta writer version 7 and reader version 3. Table protocol versions cannot be downgraded, and tables with clustering enabled are not readable by <Delta> clients that do not support all enabled Delta reader protocol table features. See [_](/versioning.md).
+
+## Choose clustering keys
+
+Clustering keys can be defined in any order. If two columns are correlated, you only need to add one of them as a clustering key.
+
+If you're converting an existing table, consider the following recommendations:
+
+| Current data optimization technique | Recommendation for clustering keys |
+| --- | --- |
+| Hive-style partitioning | Use partition columns as clustering keys. |
+| Z-order indexing | Use the `ZORDER BY` columns as clustering keys. |
+| Hive-style partitioning and Z-order | Use both partition columns and `ZORDER BY` columns as clustering keys. |
+| Generated columns to reduce cardinality (for example, date for a timestamp) | Use the original column as a clustering key, and don't create a generated column. |
+
+## Write data to a clustered table
+
+You must use a Delta writer client that supports Clustering table feature.
+
+## <a id="optimize"></a> How to trigger clustering
+
+Use the `OPTIMIZE` command on your table, as in the following example:
+
+```sql
+OPTIMIZE table_name;
+```
+
+Liquid clustering is incremental, meaning that data is only rewritten as necessary to accommodate data that needs to be clustered. Data files with clustering keys that do not match data to be clustered are not rewritten.
+
+## Read data from a clustered table
+
+You can read data in a clustered table using any <Delta> client. For best query results, include clustering keys in your query filters, as in the following example:
```

Comment:
this contradicts the statement above where I left a comment.

Reply:
need to specify minimum protocol version? (not sure how other examples address this)

Reply:
Added reader version requirements